package/src/components/Thread/__tests__/__snapshots__/Thread.test.js.snap

```diff
@@ -1127,9 +1151,14 @@ exports[`Thread should match thread snapshot 1`] = `
                                 "alignItems": "flex-end",
                                 "flexDirection": "row",
                               },
+                              [
+                                {
+                                  "marginBottom": 12,
+                                },
+                                {},
+                              ],
                               {
                                 "justifyContent": "flex-start",
-                                "marginBottom": 30,
                                 "marginTop": 0,
```

Comment:
30 to 12 change here... big change it seems.. all okay? aligned with gigma?

Reply:
Yes